bundles/org.eclipse.cdt.lsp/META-INF/MANIFEST.MF

```diff
@@ -2,14 +2,16 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.eclipse.cdt.lsp;singleton:=true
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 2.0.0.qualifier
 Export-Package: org.eclipse.cdt.lsp,
+ org.eclipse.cdt.lsp.config;x-friends:="org.eclipse.cdt.lsp.clangd",
```

Comment:
Should this have internal in the name if it is only intended to be used by friends? Or do you have 3 levels:

- Public API: no internal in names and no x-friends/x-internal 
- Internal API (between bundles): no internal in names, and x-friends/x-internal
- complete non-API: not included in Export-Package

Reply:
> Public API: no internal in names and no x-friends/x-internal
Internal API (between bundles): no internal in names, and x-friends/x-internal
complete non-API: not included in Export-Package

yes, that's what I had in mind

Reply:
`internal` segment in names comes from [this [relatively unclear] convention](https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fmisc%2Fnaming.html)